recipes-qt/automotive/qtapplicationmanager-noop.inc

```diff
@@ -0,0 +1,6 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#   SPDX-License-Identifier: MIT
+#
+
+# this file is intentionally left blank
```

Comment:
Could you add a comment about the purpose why you added this file?

Reply:
I guess it is to make the require line work in qtapplicationmanager_git.bbappend. I think it should be possible to do this in some other way, but I am not sure..

Reply:
That's correct. But I will add a comment within the file itself to explain what's going on.